aptos-move/framework/aptos-framework/sources/transaction_context.move

```diff
@@ -46,6 +51,36 @@ module aptos_framework::transaction_context {
         auid.unique_address
     }
 
+    /// Return the sender's address for the current transaction.
+    /// This function aborts if called outside of the transaction prologue, execution, or epilogue phases.
+    native fun sender_internal(): address;
+    public fun sender(): address {
+        sender_internal()
+    }
+
+    /// Return the list of the secondary signers for the current transaction.
+    /// If the current transaction has no secondary signers, this function returns an empty vector.
+    /// This function aborts if called outside of the transaction prologue, execution, or epilogue phases.
+    native fun secondary_signers_internal(): vector<address>;
+    public fun secondary_signers(): vector<address> {
+        secondary_signers_internal()
+    }
+
+    /// Return some fee payer address if a separate fee payer is specified for the current transaction.
+    /// If no separate fee payer is specified, this function returns `None`.
+    /// This function aborts if called outside of the transaction prologue, execution, or epilogue phases.
+    native fun fee_payer_opt_internal(): Option<address>;
```

Comment:
Should not be optional. If no separate fee payer is specified, sender is paying

Reply:
Renamed it to `gas_payer_internal` and changed the return type to `address` instead of `Option<address>`.